staging/src/k8s.io/client-go/util/jsonpath/jsonpath.go

```diff
@@ -297,11 +297,11 @@ func (j *JSONPath) findFieldInValue(value *reflect.Value, node *FieldNode) (refl
 	var inlineValue *reflect.Value
 	for ix := 0; ix < t.NumField(); ix++ {
 		f := t.Field(ix)
-		jsonTag := f.Tag.Get("json")
-		parts := strings.Split(jsonTag, ",")
-		if len(parts) == 0 {
+		jsonTag, found := f.Tag.Lookup("json")
+		if !found {
```

Comment:
If there is not a json tag, then the field name is the go name of the field. The field probably shouldn't be skipped.